.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
+package-lock=true
```

Comment:
Can we just get rid of this file as the default behaviour is to create a lock file?

Reply:
Some developers (like me) have a global npm config to disable `package-lock`.